dockerfiles/theia-endpoint-runtime-binary/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia:${BUILD_TAG} as builder
+
+USER root
+
+RUN cd /home/theia && \
+    yarn global add nexe@3.3.2 && nexe -v && \
+    # Build remote binary with node runtime 10.14.2 and che-theia node dependencies. nexe icludes to the binary only
+    # necessary dependencies.
+    nexe node_modules/@eclipse-che/theia-remote/lib/node/plugin-remote.js -t alpine-x64-10.16.0 -o /plugin-remote-endpoint
+
+# Light image without node. We include remote binary to this image.
+FROM alpine:3.10.2
```

Comment:
what is the difference of using FROM ALPINE:3.10.2 and using FROM scratch if we don't install anything and that the binary is self-contained ? it's due to musl linking I suppose ?

Reply:
Sorry, @benoitf, Seems, scratch has only latest tag. And this image has not `cp` and  has not shell(even without sh).  Dockerfile `RUN` section doesn't work without sh, so I can not install cp. And I think scratch has not any package manager, because it's empty image, so I don't know how to install cp or so on.

Reply:
ok I see.

I might see another problem.
We generate nexe for alpine but if the runtime is not alpine, what is the behavior ?
like using debian, centos, fedora, etc images ?

as we'll copy the bootstrap to another container ?

Reply:
@benoitf I tested with centos, fedora debian and it is works. List tested images was pretty big. You can see this list https://github.com/eclipse/che-theia/pull/267/commits/99ae354ed9586918703026eec7f0c8ce18d01267#diff-f7eaf7e082319e42b2dfd26a86ca0736R80 search by from, I commented code, but each image was tested.

Reply:
ok, still strange but if it works.